Dockerfile.full

```
@@ -90,22 +83,16 @@ RUN node -v
 
 # Install yq
 RUN curl -L https://github.com/mikefarah/yq/releases/download/v4.35.2/yq_linux_amd64 -o /usr/local/bin/yq && chmod +x /usr/local/bin/yq
-# Verify yq installation
 RUN yq --version
 
-RUN useradd github && \
-    mkdir -p /home/github && \
-    chown -R github:github /home/github && \
-    chown -R github:github /actions-runner
-
-WORKDIR /home/github
+# container home is /home/runner
+# "runner" user is created in base image, has permissions on container home
+# "docker" group is created in base image
+WORKDIR /home/runner
 
-COPY killProcess.sh ./killProcess.sh
-RUN chmod +x ./killProcess.sh
+USER runner
 
-COPY entrypoint.sh ./entrypoint.sh
+COPY --chown=runner:docker ./scripts/entrypoint.sh ./entrypoint.sh
```

```suggestion
COPY --chown=runner: ./scripts/entrypoint.sh ./entrypoint.sh
```
I think giving ownership to the user and its default groups is enough